File: lab2_fabrica/fabrica/Mascara.cs
using System;
using System.Drawing;
using System.Runtime.Serialization;

namespace fabrica
{
    [DataContract]
    public class Mascara:DecorativeCosmetics
    {
        [DataMember]
        private String brushType;
        public Mascara(String name, String brand, double price, String shade, String texture, String brushType): base(name, brand, price, shade, texture)
        {
            this.brushType = brushType;
        }
        public String Brush { 
            get => brushType;
            set => brushType = value;
        }
        public override Bitmap Img()
        {
            return (Bitmap)Bitmap.FromFile(".//img//mascara.png");
        }

        public override string getinfo()
        {
            String info =
            "Name: " + name + "\r\n" +
            "Brand: " + brand + "\r\n" +
            "Price: " + price + "$" + "\r\n" +
            "Shade: " + shade + "\r\n" +
            "Texture: " + texture + "\r\n" +
            "brushType: " + brushType + "\r\n";
            return info;
        }
    }
}

File: lab1 class/untitled/src/Mascara.java
public class Mascara extends DecorativeCosmetics{
    private String brushType;
    public Mascara(String name, String brand, double price, int shade, String scent, String texture, String brushType) {
        super(name, brand, price, shade, scent, texture);
        this.brushType = brushType;
    }
    public String getBrush(){return brushType;}
    public void setBrush(String brushType){this.brushType = brushType;}

    @Override
    public void printinfo() {
        System.out.println(name);
        System.out.println("Brand: "+ brand);
        System.out.println("Price: "+ price + "$");
        System.out.println("Scent: " + scent);
        System.out.println("Shade: " + shade);
        System.out.println("Texture: " + texture);
        System.out.println("brushType: " + brushType + "\n");
    }
}
